File: src/main/java/com/und/eventapi/repository/EventRepository.java
package com.und.eventapi.repository;

import com.und.eventapi.model.Event;
import org.springframework.data.mongodb.repository.MongoRepository;

import java.util.List;

public interface EventRepository extends MongoRepository<Event, String> {

    public List<Event> findByName(String eventName);

}

File: src/main/kotlin/com/und/eventapi/repository/EventRepository.kt
package com.und.eventapi.repository

import com.und.model.mongo.Event
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface EventRepository : MongoRepository<Event, String>, EventRepositoryUpdate {

    fun findByName(eventName: String): List<Event>

}



